oxapy/__init__.py

```
@@ -40,6 +38,8 @@ def send_file(path: str) -> Response:
     Returns:
         Response: A Response with file content
     """
+    if not os.path.exists(path):
+        raise exceptions.NotFoundError(f"The file at {path} is not found.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)